### ISSUE TITLE
Hot Wheels: Worlds Best Driver - Crashed at ingame

### ISSUE THREAD

**Issue by JohnGodgames** [citra-emu/citra#1928]
It's an old issue and it's still isn't solved. 

Unhandled exception at 0x00007FF79728BC0B in citra-qt.exe: 0xC0000094: Integer division by zero.

` size_t contiguous_input_size = config.texture_copy.size / input_width * (input_width + input_gap);
`

**Call stack:**
```
>	citra-qt.exe!GPU::Write<unsigned int>(unsigned int addr, const unsigned int data) Line 190	C++
 	citra-qt.exe!GSP_GPU::ExecuteCommand(const GSP_GPU::Command & command, unsigned int thread_id) Line 540	C++
 	citra-qt.exe!GSP_GPU::TriggerCmdReqQueue(Service::Interface * self) Line 598	C++
 	citra-qt.exe!Service::Interface::SyncRequest() Line 80	C++
 	citra-qt.exe!SVC::SendSyncRequest(unsigned int handle) Line 232	C++
 	citra-qt.exe!HLE::Wrap<&SVC::SendSyncRequest>() Line 134	C++
 	citra-qt.exe!SVC::CallSVC(unsigned int immediate) Line 1107	C++
 	citra-qt.exe!InterpreterMainLoop(ARMul_State * cpu) Line 3530	C++
 	citra-qt.exe!ARM_DynCom::ExecuteInstructions(int num_instructions) Line 87	C++
 	citra-qt.exe!Core::RunLoop(int tight_loop) Line 50	C++
 	citra-qt.exe!EmuThread::run() Line 52	C++
 	Qt5Core.dll!00000000643c7fef()	Unknown
 	[External Code]	

```

**Comment by Dragios** [comment]
@JohnGodgames Could you please test the hotfix and see if it does fix the problem.

**Comment by wwylele** [comment]
Should be fixed by #2103. If not please leave a comment and reopen this.
